client/components/follow-button/test/button.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { render } from 'enzyme';
```

Comment:
though we'd have to be a bit careful, as FollowButton is old enough that it's probably still `require`d in places and we'd have to update them to use `.default` if we added a name export to FollowButton...

maybe just leave it for now.